tools/sql/migrate_check.py

```
@@ -290,15 +357,32 @@ def main() -> None:
             ).fetchone()
             if col is None:
                 raise SystemExit(f"❌ 表 {t} 不存在或无普通列可作探针（触发器清单与 schema 脱节）")
-            probes = (f"UPDATE {t} SET {col[0]} = {col[0]} WHERE false;", f"DELETE FROM {t} WHERE false;")
-            for op_sql in probes:
+            probes = (
+                (f"UPDATE {t} SET {col[0]} = {col[0]} WHERE false;", 1 << 5),  # TRIGGER_TYPE_UPDATE
```

Ruff (0.16.1)

[error] Possible SQL injection vector through string-based query construction

(S608)